install_ant-media-server.sh

```diff
@@ -224,19 +224,23 @@ if ! [ -x "$(command -v sudo)" ]; then
   SUDO=""
 fi
 
+VERSION=$(unzip -p "$ANT_MEDIA_SERVER_ZIP_FILE" ant-media-server/ant-media-server.jar  | busybox unzip -p - | grep -a "Implementation-Version"|cut -d' ' -f2 | tr -d '\r')
+REQUIRED_VERSION="2.6"
+
 if [ "$ID" == "ubuntu" ]; then
   $SUDO apt-get update -y
-  if [ "aarch64" == $ARCH ]; then
-    $SUDO apt-get install openjdk-11-jdk unzip zip libva-drm2 libva-x11-2 libvdpau-dev -y
-    check
-  else
-    $SUDO apt-get install openjdk-11-jdk unzip zip libva-drm2 libva-x11-2 libvdpau-dev libcrystalhd-dev -y
-    check
+  $SUDO apt-get install openjdk-11-jdk unzip zip libva-drm2 libva-x11-2 libvdpau-dev -y
+  if [ "$(printf '%s\n' "$REQUIRED_VERSION" "$VERSION" | sort -V | head -n1)" != "$REQUIRED_VERSION" ]; then
```

Comment:
What if we release the version is 2.7, 3.0, 4.0? It seems that if will try to install libcrystalhd-dev packet and fail.